classes/Group.php

```diff
@@ -148,7 +148,15 @@ public static function getPriceDisplayMethod($id_group)
 
     public static function getDefaultPriceDisplayMethod()
     {
-        return Group::getPriceDisplayMethod((int)Configuration::get('PS_CUSTOMER_GROUP'));
+        return Group::getPriceDisplayMethodByGroup((int)Configuration::get('PS_CUSTOMER_GROUP'));
+    }
```

Comment:
Hello @marionf the default behavior is PS_CUSTOMER_GROUP, so if the customer group display the prices with tax excluded, then you will see all prices tax excluded whatever you're logged or not ;)